pkg/api/types.go

```diff
@@ -1920,6 +1920,9 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string
+	// If specified, the pod's tolerations
+	// +optional
+	Tolerations []Toleration
```

Comment:
I agree with @davidopp that there are very detailed comments for Taints and Tolerations correlation with their types, so I dont think thats needed here.